client/src/components/TaskStatusChart/TaskStatusChart.js

```diff
@@ -30,35 +30,38 @@ const chartLabel = ({
   );
 };
 
-const COLORS = ["#629632", "#F6BE00", "#FF0000"];
+const COLORS = ["#4F7009", "#F6BE00", "#cc0000"];
```

Comment:
I would move this constant inside the TaskStatusChart component

Reply:
Resolved: I implemented this change.